roles/network_plugin/cilium/defaults/main.yml

```
@@ -216,6 +216,9 @@ cilium_operator_extra_volume_mounts: []
 # Extra arguments for the Cilium Operator
 cilium_operator_custom_args: []
 
+# Tolerations of the cilium operator
+cilium_operator_tolerations: [{operator: "Exists"}]
```

```suggestion
cilium_operator_tolerations:
  - operator: "Exists"
```

No need for the json form, lets stay with yaml here :+1: